### DISCUSSION TITLE
GET OLDER SPONSORED BRANDS DATA THROUGH AMAZON ADS API

### DISCUSSION THREAD

**Discussion by Karthick2299** [amzn/ads-advanced-tools-docs#247] [Q&A] [answered]
Hi @elizafarley,

I created campaigns in April 2020, but now I need the data from those campaigns/sponsored brand reports. Is it possible to fetch that data through any API or is there another possible way?

Thanks in advance!

**Comment by elizafarley** [comment, marked as answer]
@Karthick2299 currently data is only available for about the last 90 days, depending on the report type. You may be able to see older data in the advertising console.

**Reply by Karthick2299** [reply]
Thanks @elizafarley .
Then how should I get my oldest campaigns data ?


I'm able to see the oldest campaign data in advertising console. But I need to fetch the data through API. Is there any possibility @elizafarley ?
